tools/tfuser/main.go

```diff
@@ -253,6 +253,10 @@ func main() {
 									Name:  "password, p",
 									Usage: "optional password",
 								},
+								cli.StringFlag{
```

Comment:
I don't think we should add this flag here.
The node ID is already specify in the provision command.
So I think the encryption needs to happen during provision command. 
- load json
- if it's a zdb reservation, load the zdb data 
- encrypt the password using the node ID
- sign the full reservation with encrypted password
- send the signed reservation to BCDB

Reply:
Yeah, I considered that first but then i realized
- The password will be unencrypted in the schema file
- The provision need to know the type of the reservation and have special handling for the zdb case.